src/views/standalone/LogsView.vue

```diff
@@ -0,0 +1,178 @@
+  errorBag.value.clear()
+  loading.value = true
+  // api call
+  ubusCall('ns.log', 'get-log', {
+    search: search.value,
+    limit: limit.value
+  })
+    .then((response: AxiosResponse<LogResponse>) => {
+      // please don't call the RegExp with an empty string, it appears to hang the system way longer than it should
+      // this STILL IS NOT a feasible solution due to the task inefficiency, but it's better than nothing
+      // TODO: defer loading and don't lock the interface
+      if (search.value == '') {
+        data.value = response.data.values
+      } else {
+        const regExStr = new RegExp(search.value, 'g')
+        data.value = response.data.values.map((entry) => {
+          return entry.replace(
+            regExStr,
+            (match) => `<span class="dark:bg-yellow-800 bg-yellow-400">${match}</span>`
+          )
+        })
+      }
+    })
+    .catch((reason: Error) => {
+      if (reason instanceof ValidationError) {
+        if (reason.errorBag.has('search')) {
+          errorBag.value.set('search', t('standalone.logs.search_error'))
+        }
+        if (reason.errorBag.has('limit')) {
+          errorBag.value.set('limit', t('standalone.logs.limit_error'))
+        }
+      } else {
+        // generic error, show in UI
+        error.value = reason
+      }
+    })
+    .finally(() => (loading.value = false))
+}
+</script>
+
+<template>
+  <NeTitle>{{ t('standalone.logs.title') }}</NeTitle>
+  <div class="space-y-8">
+    <div class="grid grid-cols-1 gap-6 sm:max-w-xl sm:grid-cols-2">
+      <NeTextInput
+        v-model="search"
+        :invalid-message="errorBag.getFirstFor('search')"
+        :label="t('standalone.logs.search')"
+        :placeholder="t('standalone.logs.search_placeholder')"
+      >
+        <template #tooltip>
+          <NeTooltip>
+            <template #content>
+              {{ t('standalone.logs.search_tooltip') }}
+            </template>
+          </NeTooltip>
+        </template>
+      </NeTextInput>
+      <NeCombobox
+        v-model="limit"
+        :invalid-message="errorBag.getFirstFor('limits')"
+        :label="t('standalone.logs.limit_rows')"
+        :options="logLimitComboboxOption"
+      />
+      <div class="col-span-1 flex gap-8 sm:col-span-2">
+        <NeToggle v-model="wrapRow" :label="t('standalone.logs.wrap_row')" />
+        <NeToggle v-model="pool" :label="t('standalone.logs.pooling')" />
+      </div>
+    </div>
+    <NeInlineNotification
+      v-if="error"
+      :description="error.message"
+      :title="t(getAxiosErrorMessage(error))"
+      kind="error"
+    />
+    <ul
+      ref="logRef"
+      class="max-h-[55vh] max-w-6xl divide-y overflow-y-scroll rounded-md border font-mono text-sm dark:divide-gray-600 dark:border-gray-600"
+    >
+      <li
+        v-for="(entry, index) in data"
+        :key="index"
+        :class="logRowStyle"
+        class="p-2 odd:bg-gray-100 odd:dark:bg-gray-950 even:dark:bg-gray-900"
```

Comment:
Missing `even:bg-gray-*` class for light theme?

Reply:
` even:dark:bg-gray-900` was the background color, removing it